### ISSUE TITLE
Auto Completion on Managed Packages

### ISSUE THREAD

**Issue by mnunezdm** [forcedotcom/salesforcedx-vscode#1177]
### Summary

_Short summary of what is going on or to provide context_.

### Steps To Reproduce:

1.  This is step 1.
1.  This is step 2. All steps should start with '1.'

### Expected result

_Describe what should have happened_.

### Actual result

_Describe what actually happened instead_.

### Additional information

_Feel free to attach a screenshot_.

**VS Code Version**:

**SFDX CLI Version**:

**OS and version**:

**Comment by lcampos** [comment]
Thanks for opening this @mnunezdm . This is a currently a limitation with the Apex language server which we're looking into.
cc @ntotten

**Comment by praksb** [comment]
@mnunezdm I agree this is an annoying bug. While we are working on making this better, as a workaround you could  temporarily set 
"namespace": "" 
in your sfdx-project.json file.

**Comment by Szandor72** [comment]
I had a feature request in that direction: ignore namespace configured in project.json for auto completion. but cannot find it anymore for the love of god. 
Thanks for the workaround!
